### PR TITLE
add Nscreen-32 support

### PR DESCRIPTION
**Pull request by simonchen007**
Add Nscreen-32 board support.

### CODE REVIEW

File: boards/nscreen-32.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  "name": "YeaCreate NSCREEN-32",
+  "upload": {
+    "flash_size": "16MB",
+    "maximum_ram_size": 327680,
+    "maximum_size": 16777216,
+    "protocols": [
+      "esptool",
+      "espota",
+      "ftdi"
+    ],
```
Comment (valeros) [inline boards/nscreen-32.json:39]: No need to explicitly specify upload protocols, they're configured in the platform dynamically.

Comment (simonchen007) [reply boards/nscreen-32.json:39]: ok, do I need to change it , and  pull request again ?

Comment (valeros) [reply boards/nscreen-32.json:39]: Simply push a new commit with a fix

Comment (simonchen007) [reply boards/nscreen-32.json:39]: ok, please close the PR, I will make a new PR again

Comment (simonchen007) [reply boards/nscreen-32.json:39]: Put it here:
https://github.com/platformio/platform-espressif32/pull/446

**Pull request closed without merging**